### DISCUSSION TITLE
part-ii/2021-102/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#418] [Question comments]
Part II 2021 Number Fields: Paper 4 , Section II, 20G 

https://questions.tripos.org/part-ii/2021-102/

**Comment by artie2000** [comment]
<details>
<summary>(a)</summary>

$\text{Cl}(K)=C_2\times C_2$

(generated by prime factors of $(2)$ and $(5)$).

$11+2\sqrt{30}$ is the fundamantal unit.

</details>
<details>
<summary>(b)</summary>

Mostly definition checking.

To show index 4 and the last bit, map $K$ to $C_2\times C_2$ by $x\to(\text{sgn}\ \sigma_1(x),\text{sgn}\ \sigma_2(x))$.

</details>
